src/controllers/organizationController.ts

```diff
@@ -104,61 +110,24 @@ export default class OrganizationController {
           if (isSome(maybeResponse)) {
             return maybeResponse.value;
           }
-          const errorResponseOrSuccessResponse = await registerOrganization(
+          const errorResponseOrSuccessResponse = await createOnboardingRequests(
             organizationRegistrationParams,
             user
-          );
-          return errorResponseOrSuccessResponse.map(async response => {
-            const organization = response.payload;
-            // TODO:
-            //  the documents must be stored on cloud (Azure Blob Storage).
-            //  @see https://www.pivotaltracker.com/story/show/169644958
-            const outputFolder = `./documents/${organization.ipa_code}`;
-            try {
-              await fs.promises.mkdir(outputFolder, { recursive: true });
-              const arrayOfMaybeError = await Promise.all([
-                this.documentService.generateDocument(
-                  localeIt.organizationController.registerOrganization.contract.replace(
-                    "%s",
-                    `${organization.name} ${organization.fiscal_code}`
-                  ),
-                  `${outputFolder}/contract.pdf`
-                ),
-                this.documentService.generateDocument(
-                  // TODO:
-                  //  refactor this operation using an internationalization framework allowing params interpolation in strings.
-                  //  @see https://www.pivotaltracker.com/story/show/169644146
-                  localeIt.organizationController.registerOrganization.delegation
-                    .replace(
-                      "%legalRepresentative%",
-                      `${organizationRegistrationParams.legal_representative.given_name} ${organizationRegistrationParams.legal_representative.family_name}`
-                    )
-                    .replace("%organizationName%", organization.name)
-                    .replace(
-                      "%delegate%",
-                      `${user.givenName} ${user.familyName}`
-                    ),
-                  `${outputFolder}/mandate-${user.fiscalCode.toLowerCase()}.pdf`
-                )
-              ]);
-              const someError = arrayOfMaybeError.find(isSome);
-              if (someError) {
-                log.error(someError.value);
-                return ResponseErrorInternal("Internal server error");
-              }
-              return response;
-            } catch (error) {
-              log.error(error);
-              return ResponseErrorInternal("Internal server error");
-            }
-          }).value;
+          ).run();
+          return errorResponseOrSuccessResponse.map(
+            async successResponse =>
+              (await this.createOnboardingDocuments(
+                organizationRegistrationParams.ipa_code,
+                successResponse
+              ).run()).value
```

Comment:
I expect
```ts
return tryCatch(()=> createOnboardingRequests(...))
  .chain(successResponse => this.createOnboardingDocuments(...)).run()
```

`run` should be called once at the end of the task chain. Ideally `run` should never be called (this is a limitation in how `toExpressHandler` works atm)